### ISSUE TITLE
Security

### ISSUE THREAD

**Issue by michalszynkiewicz** [project-ncl/license-dictionary#7]
Secure adding, editing and importing licenses

**Comment by thescouser89** [comment]
We could add security via Keycloak I guess

**Comment by michalszynkiewicz** [comment]
yes, this is the idea. It is quite easy to add keycloak with Swarm.
The difficult part would be the UI :D
But it can be split into two tasks, securing the backend is more important
